app/appBase.go

```diff
@@ -309,3 +234,32 @@ func filterItemsByPinned(clipboardItems []config.ClipboardItem, isPinned bool) [
 
 	return filteredItems
 }
+
+func pinnedStyle() string {
+	var color string
```

Comment:
I would use this to remove the else


```go
func pinnedStyle() string { 
         pinChar := " " 
         config := config.GetTheme() 
  
         color := "#FF0000" 
         if config.UseCustom { 
                 color = config.PinIndicatorColor 
         } 
         return lipgloss.NewStyle().Foreground(lipgloss.Color(color)).SetString(pinChar).Render() 
 }
```

Reply:
Thank you as always :pray:  I'm still getting used to the recommended Go design patterns.